src/Particular.CodeRules/ConfigureAwaitAnalyzer/ConfigureAwaitAnalyzer.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Collections.Immutable;
+
+namespace Particular.CodeRules.ConfigureAwaitAnalyzer
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class ConfigureAwaitAnalyzer : DiagnosticAnalyzer
+    {
+        public const string RuleIdentifier = "PCR0001";
+        private static DiagnosticDescriptor ruleDescriptor = new DiagnosticDescriptor(RuleIdentifier, "Await used without specifying ConfigureAwait", "Await used without specifying ConfigureAwait", "Usage", DiagnosticSeverity.Warning, true);
+
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
```

Comment:
make this an expression body 

    public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(ruleDescriptor);